app/models/enrollment.rb

```diff
@@ -38,6 +38,9 @@ class Enrollment < ApplicationRecord
       user&.add_role(event.as_personified_event.to_sym, enrollment)
 
       Enrollment::SendMailJob.perform_now(enrollment, user, event)
+      if event == 'send_application'
```

Comment:
To my opinion defining a special action in the send_application transition may be clearer : 

```rb
after_transition :pending => :sent do |enrollment, transition|
   Enrollment::SendMailJob.perform_now(enrollment, user, 'send_application_feedback')
end
```